src/NUnitTestAdapterInstall/source.extension.vsixmanifest

```diff
@@ -3,10 +3,10 @@
   <Identifier Id="NUnitTestAdapterInstall..7c53286e-ac4f-485f-915c-5ec5a4c47b0c">
     <Name>NUnit Test Adapter</Name>
     <Author>Charlie Poole</Author>
-    <Version>1.2.0.1</Version>
-    <Description xml:space="preserve">NUnit adapter for integrated test execution under Visual Studio 2012 (all updates), Visual Studio 2013 (all updates), and the Visual Studio 2015 Preview and CTPs. Compatible with NUnit 2.0 through 2.6.3.</Description>
+    <Version>1.9.9.2</Version>
```

Comment:
Yes.  This manifest file looks much older, the Description seems outdated.  Sure this is a recent one ?